probe_scraper/runner.py

```diff
@@ -78,21 +79,20 @@ def write_moz_central_probe_data(probe_data, revisions, out_dir):
         dump_json(channel_probes, data_dir, "all_probes")
 
 
-def write_external_probe_data(repo_data, out_dir):
-    # Save all our files to "outdir/<repo>/..." to mimic a REST API.
+def write_glean_metric_data(repo_data, out_dir):
+    # Save all our files to "outdir/glean/<repo>/..." to mimic a REST API.
     for repo, probe_data in repo_data.items():
-        base_dir = os.path.join(out_dir, repo)
+        base_dir = os.path.join(out_dir, "glean", repo)
 
         print("\nwriting output:")
-        dump_json(general_data(), base_dir, "general")
 
-        data_dir = os.path.join(base_dir, "mobile-metrics")
-        dump_json(probe_data, data_dir, "all_probes")
+        dump_json(general_data(), base_dir, "general")
+        dump_json(probe_data, base_dir, "metrics")
```

Comment:
Removes a bit of nesting, now we'll have:
```
/glean/fenix/metrics
/glean/fenix/general
/glean/reference-browser/metrics
/glean/reference-browser/general
```

Reply:
Would you mind documenting this in the [readme](https://github.com/mozilla/probe-scraper#accessing-the-data-files)? I think the `README` requires an update for both the file format and the REST-like API format for glean data.